src/components/skills-quiz/GoalDropdown.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { Form } from '@edx/paragon';
+import PropTypes from 'prop-types';
+import {
+  DROPDOWN_OPTION_CHANGE_CAREERS, DROPDOWN_OPTION_CHANGE_RULE, DROPDOWN_OPTION_GET_PROMOTED, DROPDOWN_OPTION_OTHER,
+} from './constants';
+
+const GoalDropdown = ({ handleGoalOptionChange }) => {
+  const [currentGoal, setCurrentGoal] = useState('Goal');
+  const handleDropdownChange = e => {
+    setCurrentGoal(e.target.value);
+    handleGoalOptionChange(e.target.value);
+  };
+  return (
+    <Form.Control
+      as="select"
+      value={currentGoal}
+      onChange={handleDropdownChange}
+      floatingLabel="Goal"
+    >
+      <option value="">Select a goal</option>
+      <option>{DROPDOWN_OPTION_CHANGE_CAREERS}</option>
```

Comment:
Except of adding all the options one by one manually,  can we please create a dictionary of these in constants and then append them here using a for loop. That will increase readability and also it will be easy to add and remove goals.

Reply:
@zamanafzal I had discussed map approach with @saleem-latif but he recommended this existing approch.